### PR TITLE
`New Post` - New patterns for July 2022

### PR DESCRIPTION
**Pull request by argyleink**
Announcement / hype post for new patterns in #7898. 

Currently in draft while the patterns PR lands and the embedded `CodePattern`s can be matched; mixins are in the post but don't resolve and it makes the post look short/incomplete. 

https://deploy-preview-8003--web-dev-staging.netlify.app/new-patterns-june-2022/

to do:
- [x] update date meta to release date
- [x] test pattern embeds after #7898 lands

### CODE REVIEW

**Comment by argyleink** [comment]
ready for review 👍🏻

File: src/site/content/en/blog/new-patterns-june-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: New Patterns
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-patterns-june-2022/index.md:3]:
```suggestion
title: New patterns
```

File: src/site/content/en/blog/new-patterns-june-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+subhead: Animations, theming, components and more layout patterns are live and ready to help kick start or inspire your UI and UX.
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-patterns-june-2022/index.md:4]:
```suggestion
subhead: Animations, theming, components,and more layout patterns are live and ready to help kick start or inspire your UI and UX.
```

File: src/site/content/en/blog/new-patterns-june-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+authors:
+  - adamargyle
+description: Animations, theming, components and more layout patterns are live and ready to help kick start or inspire your UI and UX.
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-patterns-june-2022/index.md:7]:
```suggestion
description: Animations, theming, components, and more layout patterns are live and ready to help kick start or inspire your UI and UX.
```

File: src/site/content/en/blog/new-patterns-june-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+date: 2022-06-13
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-patterns-june-2022/index.md:8]: update to release date

File: src/site/content/en/blog/new-patterns-june-2022/index.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+other is blue.", width="360", height="240" %}
+
+View [the theming patterns landing page](/patterns/theming/) or check each out
+individually:
+
+1. [Color Schemes](/patterns/theming/color-schemes/)
+1. [Theme Switch](/patterns/theming/theme-switch/)
+
+One pattern is for building a client side [theme
+switch](/building-a-theme-switch-component/) so user's can indicate their
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-patterns-june-2022/index.md:116]:
```suggestion
switch](/building-a-theme-switch-component/) so users can indicate their
```

**Review by rachelandrew — APPROVED** [review]
Just a couple of nits - and don't forget to update the date.

File: src/site/content/en/blog/new-patterns-june-2022/index.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: New patterns
+subhead: Animations, theming, components,and more layout patterns are live and ready to help kick start or inspire your UI and UX.
```
Comment (argyleink) [inline src/site/content/en/blog/new-patterns-june-2022/index.md:4]:
```suggestion
subhead: Animations, theming, components, and more layout patterns are live and ready to help kick start or inspire your UI and UX.
```

**Pull request merged**